frontend/app/(profile)/profile/projects/[project_id]/page.tsx

```
@@ -298,6 +283,145 @@ export default function ProjectDetails() {
+          {/* Assigned Freelancer */}
+          {project.status === "Assigned" && assignment && (
+            <div className="bg-white p-6 rounded-lg shadow-sm border border-gray-100 hover:shadow-md transition-shadow">
+              <h3 className="text-xl font-semibold mb-4 text-gray-800 border-b pb-2">
+                Assignment Details
+              </h3>
+
+              <div className="grid grid-cols-1 md:grid-cols-2 gap-6">
+                {/* Freelancer Information */}
+                <div className="bg-blue-50 p-4 rounded-md">
+                  <h4 className="font-semibold text-blue-800 mb-3">
+                    Assigned Freelancer
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)